src/Notes/WC_Admin_Notes_Onboarding_Payments.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * WooCommerce Admin: Payments reminder note.
+ *
+ * Adds a notes to complete the payment methods.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+use \Automattic\WooCommerce\Admin\Features\Onboarding;
+
+/**
+ * WC_Admin_Notes_Onboarding_Payments.
+ */
+class WC_Admin_Notes_Onboarding_Payments {
+	const NOTE_NAME = 'wc-admin-onboarding-payments-reminder';
+
+	/**
+	 * Creates a note to remind store owners to set up payments.
+	 */
+	public static function possibly_add_onboarding_payments_note() {
+		// This note should only be added if the task list is still shown.
+		if ( ! Onboarding::should_show_tasks() ) {
+			return;
+		}
+
+		// Make sure payments task was skipped at least 3 days ago.
+		$three_days_in_seconds = 3 * DAY_IN_SECONDS;
+		$payments_task         = get_option( 'woocommerce_task_list_payments', array() );
+		if (
+			! isset( $payments_task['skipped'] ) ||
+			( time() - $payments_task['timestamp'] ) < $three_days_in_seconds
```

Comment:
Is the logic correct here? 

age = current time - skipped time

age is _less_ than three days

Should that be _more_ than three days?

Reply:
if I run this without skipping the Payments task (and just ignore it) the `woocommerce_task_list_payments` option doesn't get created and I get this:

```
PHP Notice:  Undefined index: timestamp in /srv/www/wordpress-two/public_html/wp-content/plugins/woocommerce-admin/src/Notes/WC_Admin_Notes_Onboarding_Payments.php on line 40
```

Reply:
>Should that be more than three days?

I believe the current logic is correct.  If it's less than 3 days, we return and don't add the note.  If it's more than 3 days we want to add it, so we don't return.

>if I run this without skipping the Payments task (and just ignore it) the woocommerce_task_list_payments option doesn't get created and I get this:

Hmm, that's an interesting one.  I think the only way you would hit `Undefined index: timestamp` is if you had already skipped and the  `skipped` index exists; otherwise it should never get evaluated.

It may be that you skipped prior to this PR, so there is `skipped` but no `timestamp`.  I added a check to see if this index is set and return early if not.  Can you let me know if that fixes the error for you @becdetat?